python/add_corona.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+#===================================================
+#
+#    Copyright (c) 2025
+#      SMASH Hybrid Team
+#
+#    GNU General Public License (GPLv3 or later)
+#
+#===================================================
+
+import os
+import sys
+import argparse
+import numpy as np
+
+def read_corona_file(file_path):
+    with open(file_path, 'r') as f:
+        lines = [line.strip().split() for line in f if not line.startswith('#')]
+    array = np.atleast_2d(np.array(lines, dtype=str)) if len(lines) != 0 else []
+    return array
+
+def initialize_final_file(file_path):
+    header_string = "#!OSCAR2013 particle_lists t x y z mass p0 px py pz pdg ID charge \n\
+# Units: fm fm fm fm GeV GeV GeV GeV GeV none none e \n\
+# SMASH-3.3 \n"
+    with open(file_path, 'w') as f:
+        f.write(header_string)
+
+def read_Sampler_file(file_path, file_path_new, array_to_append):
```

Comment:
I'm not sure if we have a convention for it, but I'd be in favor of all lower-case function name since all the others are.

```suggestion
def read_sampler_file(file_path, file_path_new, array_to_append):
```